src/persist/src/indexed/mod.rs

```diff
@@ -281,6 +285,11 @@ impl<K: Data, V: Data, U: Buffer, L: Blob> Indexed<K, V, U, L> {
             while snap.read(&mut updates) {}
         }
 
+        // Trace batches are required to be sorted by (k, v, ts).
```

Comment:
theres a couple of TODOs that we can get rid of now e.g.

```
103         // TODO: Sort the updates in the batch by `(key, value, time)` (or
104         // ensure that they're sorted, if it turns out this work should have
105         // happened somewhere else).
```

in `src/persist/src/indexed/trace.rs`

Reply:
Nice! Done!